CourseController/src/views/admin/AdminModules.vue

```diff
@@ -0,0 +1,42 @@
+<script>
+import AdminNavbar from "../../components/AdminNavbar.vue";
+import AdminModule from "../../components/AdminModule.vue";
+import {provide, ref} from "vue";
+
+export default {
```

Comment:
So we don't have to do this.
This is the old way of doing it, it does the same thing, it's just harder to maintain it if we use both ways simultaneously